models/object_detection_yolox/yolox.py

```diff
@@ -63,13 +63,11 @@ def postprocess(self, outputs):
         max_scores = np.amax(scores, axis=1)
         max_scores_idx = np.argmax(scores, axis=1)
 
-        # batched-nms, TODO: replace with cv2.dnn.NMSBoxesBatched when OpenCV 4.7.0 is released
-        max_coord = boxes_xyxy.max()
-        offsets = max_scores_idx * (max_coord + 1)
-        boxes_for_nms = boxes_xyxy + offsets[:, None]
-        keep = cv2.dnn.NMSBoxes(boxes_for_nms.tolist(), max_scores.tolist(), self.confThreshold, self.nmsThreshold)
+        keep = cv2.dnn.NMSBoxesBatched(boxes_xyxy.tolist(), max_scores.tolist(), max_scores_idx.tolist(), self.confThreshold, self.nmsThreshold)
```

Comment:
Please add a note in the readme saying it requires opencv>=4.7.

Reply:
OK